bindings/node/src/mongocrypt.cc

```diff
@@ -674,6 +674,43 @@ NAN_METHOD(MongoCrypt::MakeDataKeyContext) {
         return;
     }
 
+    if (info[1]->IsObject()) {
+        v8::Local<v8::Object> options = Nan::To<v8::Object>(info[1]).ToLocalChecked();
+        auto ALT_NAMES_KEY = Nan::New("keyAltNames").ToLocalChecked();
+        if (Nan::Has(options, ALT_NAMES_KEY).FromMaybe(false)) {
+            v8::Local<v8::Value> keyAltNames = Nan::Get(options, ALT_NAMES_KEY).ToLocalChecked();
+
+            if (!(keyAltNames->IsNull() || keyAltNames->IsUndefined())) {
+                if (!keyAltNames->IsArray()) {
+                    Nan::ThrowTypeError("keyAltNames must be an array of strings");
+                    return;
+                }
+
+                v8::Local<v8::Array> keyAltNamesArray = v8::Local<v8::Array>::Cast(keyAltNames);
```

Comment:
Since you're using `Nan::To` everywhere else, for consistency?

```suggestion
                v8::Local<v8::Array> keyAltNamesArray = Nan::To<v8::Array>(keyAltNames);
```

Reply:
I'm getting a compiler error with that change: 
```
./src/mongocrypt.cc:688:53: error: no matching function for call to 'To'
            v8::Local<v8::Array> keyAltNamesArray = Nan::To<v8::Array>(keyAltNames);
                                                    ^~~~~~~~~~~~~~~~~~
../node_modules/nan/nan_converters.h:61:38: note: candidate template ignored: substitution failure [with T = v8::Array]: implicit instantiation of undefined template 'Nan::imp::ToFactory<v8::Array>'
typename imp::ToFactory<T>::return_t To(v8::Local<v8::Value> val) {
```
I also added `.ToLocalChecked()` but the error is still there, not sure what is different from the other line where we use this:
```cxx
v8::Local<v8::Object> options = Nan::To<v8::Object>(info[1]).ToLocalChecked();
```

Reply:
Ok – a little surprised that Nan only provides this for some types, but let’s just leave this as it is then :)

Reply:
Super, yea that is unexpected but TIL, thanks for your help!